cmd/telliot/main.go

```diff
@@ -322,23 +320,23 @@ func mineCmd(logSetup func(string) log.Logger) func(*cli.Cmd) {
 					exitChannels = append(exitChannels, &ch)
 
 					var err error
-					ds, err = ops.CreateDataServerOps(ctx, logger, ch)
-					ExitOnError(err, "creating data server")
-
+					ds, err = ops.CreateDataServerOps(ctx, logger, database, proxy, clt, cont, acc, ch)
```

Comment:
Where is the `ctx` here comming from? Is it another global variable that will be removed in the cli PR or if not should pass it as an argument across the functions.

Reply:
It's coming from a global variable, but I think it could be created in this function, as it's not used anywhere before that